scripts/noshake.json

```diff
@@ -408,6 +408,8 @@
   ["Mathlib.Analysis.CStarAlgebra.ContinuousFunctionalCalculus.Unital"],
   "Mathlib.AlgebraicTopology.DoldKan.Notations":
   ["Mathlib.AlgebraicTopology.AlternatingFaceMapComplex"],
+  "Mathlib.AlgebraicGeometry.PrimeSpectrum.Basic":
+  ["Mathlib.Tactic.StacksAttribute"],
```

Comment:
Shouldn't `Tactic.StacksAttribute` be made one of those global noshakes? See `ignoreImport` at the top of this file

Reply:
Sounds good to me.